packages/docker.rb

```diff
@@ -0,0 +1,177 @@
+# Adapted from Arch Linux docker PKGBUILD at:
+# https://github.com/archlinux/svntogit-community/raw/packages/docker/trunk/PKGBUILD
+
+require 'package'
+
+class Docker < Package
+  description 'Pack, ship and run any application as a lightweight container'
+  homepage 'https://www.docker.com/'
+  version '24.0.1'
+  license 'Apache'
+  compatibility 'x86_64 aarch64 armv7l'
+  source_url 'https://github.com/docker/cli.git'
+  git_hashtag "v#{version}"
+
+  binary_url({
+    aarch64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/docker/24.0.1_armv7l/docker-24.0.1-chromeos-armv7l.tar.zst',
+     armv7l: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/docker/24.0.1_armv7l/docker-24.0.1-chromeos-armv7l.tar.zst',
+     x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/docker/24.0.1_x86_64/docker-24.0.1-chromeos-x86_64.tar.zst'
+  })
+  binary_sha256({
+    aarch64: '923b0eb95c947dde422e196abd0f1ddbc0958792db2807e2c55c3048b700702a',
+     armv7l: '923b0eb95c947dde422e196abd0f1ddbc0958792db2807e2c55c3048b700702a',
+     x86_64: '21f9db6ec0086ab6452be9cd50a6bc980eb00caafb439a3387802c5363085ec1'
+  })
+
+  depends_on 'bridge_utils'
+  depends_on 'btrfsprogs' => :build
+  depends_on 'containerd'
+  depends_on 'elogind' => :build
+  depends_on 'eudev'
+  depends_on 'glibc' # R
+  depends_on 'go' => :build
+  depends_on 'go_md2man' => :build
+  depends_on 'iproute2'
+  depends_on 'lvm2' # R
+  depends_on 'sqlite'
+
+  no_env_options
+  no_fhs
+
```

Comment:
Add `def self.preflight` section to determine compatibility?  IOW, abort and display an error message to Chromebook users that do not have support for Linux Apps?

Reply:
Done!

Reply:
Simple enough!